src/main/java/org/candlepin/subscriptions/db/model/Subscription.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.db.model;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+
+/**
+ * Subscription entities represent data from a Candlepin Pool
+ */
+@Entity
+@Table(name = "subscription")
+@IdClass(Subscription.SubscriptionId.class)
+public class Subscription {
+
+    @Id
+    @Column(name = "sku")
+    private String sku;
+
+    @Id
+    @Column(name = "owner_id")
+    private String  ownerId;
+
+    @Column(name = "subscription_id")
+    private String subscriptionId;
```

Comment:
`subscriptionId` should be the Id, actually. See comment on liquibase changeset for rationale.

Reply:
updated